dhl_sdk/entities.py

```diff
@@ -339,6 +343,7 @@ def predict(
         timestamps: list,
         inputs: dict,
         timestamps_unit: str = "s",
+        model_confidence: float = 80,
```

Comment:
how will we generally handle more inputs and changes to the PredictionConfig ? you are adding model_confidence here which is probably smth that will stick, but what about other parameters (like startingIndex, or the conditional feeding configurations, where do want to put those?)

we should think of a proper concept before we just keep adding input parameters to this signature